draft-ietf-rats-corim.md

```diff
@@ -2319,7 +2319,36 @@ groups to use this information as they see fit".
 
 # Security and Privacy Considerations {#sec-sec}
 
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/11
+Evidence appraisal is at the core of any RATS protocol flow, mediating all interactions between Attesters and their Relying Parties.
+The Verifier is effectively part of the Attesters' and Relying Parties' trusted computing base (TCB).
+Any mistake in the appraisal process could have security implications.
+For instance, it could lead to the subversion of an access control function, creating a chance for privilege escalation.
+
+Therefore, the Verifier’s code and configuration, especially those of the CoRIM processor, are primary security assets that must be built and maintained as securely as possible.
+
+The protection of the Verifier system should be considered throughout its entire lifecycle, from design to operation.
+This includes the following aspects:
+
+- Minimizing implementation complexity (see also {{Section 6.1 of -rats-endorsements}});
+- Using memory-safe programming languages;
+- Using secure defaults;
+- Minimizing the attack surface by removing unnecessary features that could be exploited by attackers;
+- Applying the principle of least privilege to the system's users;
+- Minimizing the potential impact of security breaches by implementing separation of duties in both the software and operational architecture;
+- Conducting regular, automated audits and reviews of the system, such as ensuring that users' privileges are correctly configured and that any new code has been audited and approved by independent parties;
+- Failing safely in the event of errors to avoid compromising the security of the system.
+
+The appraisal process should be auditable, reproducible, and transparent.
+The integrity of the code and data during execution should be made an explicit objective, for example ensuring that the appraisal functions are computed in a trusted execution environment (TEE).
+
+The integrity of public key material and the secrecy of private key material must be ensured at all times.
```

Comment:
This section is quite good, however I would certainly highlight the importance of the secrecy of the private key material used to sign the `Attestation Results` originating from a Verifier, as a fundamental item to be highlighted here.

Reply:
no, CoRIM doesn't deal with ARs

Reply:
I thought we were also explaining the auditability of the Verifier code here?

Reply:
`The appraisal process should be auditable, reproducible, and transparent.`